### ISSUE TITLE
Please update lwjgl to latest lwjgl-2.9.4-nightly

### ISSUE THREAD

**Issue by og1996** [Beyond-Reality/BeyondRealityModPack#208]
TL;DR The version in Beyond Reality's bin folder is 2.9.1; the latest release is currently 2.9.4 (see http://forums.atlauncher.com/thread-676-post-5094.html for more info). Having the older version causes random Minecraft crashes on my machine and lowers FPS/lag. The 2.9.4 version downloads with Minecraft 1.8, but is fully compatible with 1.7.10. It can be downloaded from http://legacy.lwjgl.org/ but I think ATLauncher provides an automatic way to upgrade the version.

More info: for the longest time, I've been having random crashes with 1.7.10 Minecraft packs...just a random, instant crash to desktop along with a useless NVIDIA error #8 message. I searched high and low on Minecraft forums for the root cause of this, reinstalling drivers, etc., but never found a solution (and running aromabackup to save my corrupted worlds). Not long ago, I was running Minecraft in debugging mode due to an issue in ATLauncher and caught the elusive error in the log screen: "OpenGL: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.". After searching for that phrase, I ended up with a lot of useless reinstall driver posts and a single post on MCF suggesting to update lwjgl to the latest version. After replacing the bin\lwjgl* and bin\natives\lwjgl* files with those in the MC 1.8 bin folder, these issues have completely gone away. My FPS has also improved a good 15~20% and I'm getting lower world-gen lag on new worlds. Woohoo all around.

**Comment by Dyonovan** [comment]
So thats whats causing it. I will update for 2.2.5 that I am working on now

**Comment by Dyonovan** [comment]
Ok just a quick update. Unfortunately LWJGL is one of the files that ATLauncher automatically does. I have no way to change it for the pack as its hardcoded. All I can suggest is to update it manually yourself.
